backend/sample_data/database_template.json5

```diff
@@ -85,35 +89,22 @@
             ],
             "description": "Is read consistency (any read request will return the most recent write) one of the most important requirements?"
```

Comment:
"one of the most important requirements?" -> từ này dễ gây lack of information vì nếu user k nói thẳng ra thì cũng khó suy luận dựa trên dữ liệu có sẵn